FogCreek/Glitch-Community
# Plump clave: combine collection, category, featured collection into single component

## Links
* Remix link: https://plump-clave.glitch.me/

## GIF/Screenshots:
![Screen Shot 2019-06-18 at 5 38 50 PM](https://user-images.githubusercontent.com/938722/59721790-190d6e80-91f0-11e9-9bea-d752abae8b6e.png)
![Screen Shot 2019-06-18 at 5 39 07 PM](https://user-images.githubusercontent.com/938722/59721791-190d6e80-91f0-11e9-8ffe-0c5eb7deff9f.png)
![Screen Shot 2019-06-18 at 5 39 27 PM](https://user-images.githubusercontent.com/938722/59721792-190d6e80-91f0-11e9-860f-5778c2ea16cb.png)
![Screen Shot 2019-06-18 at 5 39 42 PM](https://user-images.githubusercontent.com/938722/59721793-19a60500-91f0-11e9-896b-89cb73bba84c.png)

## Changes:
* create CollectionContainer component
* replace Collection page body with CollectionContainer
* replace Category page body with CollectionContainer, and adapt category data to match collection shape
* replace FeaturedCollections with list of CollectionContainers with `preview` flag
 
## How To Test:
* log into https://plump-clave.glitch.me/
* verify the featured collections on the home page render correctly
* verify category page https://plump-clave.glitch.me/hello-worlds renders
* create a collection
* the collection page should render, and you should be able to add projects to it
